mbtiles/src/mbtiles.rs

```diff
@@ -57,6 +57,10 @@ pub struct Statistics {
     pub schema: MbtType,
     pub page_size: u64,
     pub level_details: Vec<LevelDetail>,
+    pub count: u64,
```

Comment:
all these should be `Option<u64>` / `Option<f64>`

Reply:
Oops Sure

Reply:
`Largest` and` smallest` has been Optional, but I kept `count` and `average` still as the returned type of these are not optional, if the vec is empty, they just return zero.
`let count = level_details.iter().map(|l| l.count).sum();`  
`let average = level_details
                .iter()
                .map(|l| l.average * l.count as f64)
                .sum::<f64>()`